src/expression_utils.jl

```diff
@@ -0,0 +1,172 @@
+#Returns the length of a expression tuple, or 1 if it is not an expression tuple (probably a  Symbol/Numerical).
+function tup_leng(ex::ExprValues)
+    (typeof(ex)==Expr && ex.head == :tuple) && (return length(ex.args))
+    return 1
+end
+
+#Gets the i'th element in a expression tuple, or return the input itself if it is not an expression tuple (probably a  Symbol/Numerical).
+function get_tup_arg(ex::ExprValues,i::Int)
+    (tup_leng(ex) == 1) && (return ex)
+    return ex.args[i]
+end
+
+function splitplus!(ex)
+    dosplit = ex.head == :(=) && ex.args[2] isa Expr && ex.args[2].head == :call && ex.args[2].args[1] == :(+)
+    if dosplit
+      summands = ex.args[2].args[2:end]
+      ex.args[2] = foldl((x,y)->(:(($x + $y))), summands)
+    end
+    dosplit
+  end
+  
+  #Creates an expression which can be evaluated to an actual function. Input is an array of expression were each entry is a line in the function. Uses the array of expressions generated in either get_f or get_g.
+  function make_func(func_expr::Vector{Expr},reactants::OrderedDict{Symbol,Int}, parameters::OrderedDict{Symbol,Int})
```

Comment:
indent

Reply:
Yeah, I fixed this in the other PR. vscode doesn't like to preserve indentation when copying and pasting for some reason...